zipline/pipeline/loaders/blaze/dividends.py

```diff
@@ -6,12 +6,12 @@
     SID_FIELD_NAME,
     TS_FIELD_NAME,
 )
-from zipline.pipeline.data.dividends import (
+from ...data import (
     DividendsByExDate,
     DividendsByAnnouncementDate,
     DividendsByPayDate
 )
-from zipline.pipeline.loaders.dividends import (
+from ...loaders import (
```

Comment:
I think that two levels up should be the max for relative imports, after that I think it stops helping the readability.